src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <ImplicitlyExpandNETStandardFacades>false</ImplicitlyExpandNETStandardFacades>
```

Comment:
>ImplicitlyExpandNETStandardFacades [](start = 5, length = 34)

why?

Reply:
this build task caused a warning "resolved file has bad image" since we take dependency on another nuproj project

---
In reply to: [258436912](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/50#discussion_r258436912) [](ancestors = 258436912)